README.md

```diff
@@ -94,6 +94,14 @@ Putting it all together, it is 6 variant test cases.
 - Applying the `Web variant` to a `Standard Web project`
 - Applying the `Web variant` to a `Custom Web project`
 
+### Release
+
+The release process is following the [Git flow](https://nimblehq.co/compass/development/version-control/#releases-).
+
+Once a `release/<version number>` is created, to publish the new version to https://hex.pm/packages/nimble_phx_gen_template, we need to increase the version number in the `mix.ex` file on the release branch before the merge action.
+
+Once the release branch is merged into the master, Github Action will publish the template to https://hex.pm/packages/nimble_phx_gen_template automatically.
```

Comment:
```suggestion
Once the release branch is merged into the master, Github Action will automatically publish the template to https://hex.pm/packages/nimble_phx_gen_template.
```

Reply:
Fixed in https://github.com/nimblehq/elixir-templates/pull/63/commits/421a87bde1c932dfd3301dde28b40f7a95a697be @hanam1ni